File: src/main/java/com/ggemo/bilidanmakuclient/http/response/impl/RoomInitResponse.java
package com.ggemo.bilidanmakuclient.http.response.impl;

import com.alibaba.fastjson.JSON;
import com.ggemo.bilidanmakuclient.http.response.AbstractResponse;
import com.ggemo.bilidanmakuclient.http.response.responsedataa.RoomInitResponseData;
import lombok.Data;

public class RoomInitResponse extends AbstractResponse<RoomInitResponseData> {
    public RoomInitResponse(int code, String message, RoomInitResponseData data) {
        super(code, message, data);
    }

    public static RoomInitResponse parse(String json) {
        RoomInitResponse res = JSON.parseObject(json, RoomInitResponse.class);
        return res;
    }
}

File: blive-danmaku-client-jvm/src/main/kotlin/com/ggemo/va/blivedanmakuclient/httpclient/httpresponse/impl/RoomInitResponse.kt
package com.ggemo.va.bilidanmakuclient.http.response.impl

import com.ggemo.va.bilidanmakuclient.http.response.responsedataa.RoomInitData
import com.ggemo.va.bilidanmakuclient.http.response.AbstractResponse
import com.alibaba.fastjson.JSON

class RoomInitResponse :
    AbstractResponse<RoomInitData?>() {
    companion object {
        fun parse(json: String?): RoomInitResponse {
            return JSON.parseObject(json, RoomInitResponse::class.java)
        }
    }
}